SECURITY.md

```diff
@@ -0,0 +1,44 @@
+# Desmos Security Policy
+
+The Desmos core team and community takes all security issues and vulnerabilities very seriously.
+
+Thanks for improving the security of Desmos. We appreciate your efforts. Following these responsible disclosure guidelines will make sure your contribution is acknowledged.
+
+To report a security issue go to [Discord](https://discord.desmos.network) and alert the core engineers:  
+* Riccardo | Desmo `Riccardo Montagnin#5414`
+* Paul | Desmos `Paul | Desmos#0380`
+* Manuel | Desmos `Manuel Turetta#8033`
+
+
+_Please avoid opening a public Github issue or posting on social media_.
+
+The Desmos team will respond with the next steps following the email. The team will keep you informed on the remediation process and may ask for additional guidance/information.
```

Comment:
```suggestion
The Desmos team will respond with the next steps following the report. It will also keep you informed on the remediation process and may ask for additional guidance/information.
```